docs/introduction/packages.md

```diff
@@ -49,26 +50,60 @@ kind: Provider
 metadata:
   name: provider-gcp
 spec:
+  crossplane:
+    version: ">=v1.0.0"
   controller:
-    image: crossplane/provider-gcp-controller:master
+    image: crossplane/provider-gcp-controller:v0.14.0
+    permissionRequests:
+    - apiGroups:
+      - apiextensions.crossplane.io
+      resources:
+      - compositions
+      verbs:
+      - get
+      - list
+      - create
+      - update
+      - patch
+      - watch
 ```
 
+See all available fields in the [official documentation][provider-docs].
+
 > Note: The `meta.pkg.crossplane.io` group does contain custom resources that
 > may be installed into the cluster. They are strictly used as metadata in a
 > Crossplane package.
 
-The `spec.controller.image` fields specifies that the `Provider` desires for a
-`Deployment` to be created with the provided image. It is important to note that
-this image is separate from the package image itself. In the case above, it is
-an image containing the `provider-gcp` controller binary.
-
 A Provider package may optionally contain one or more CRDs. These CRDs will be
 installed prior to the creation of the Provider's `Deployment`. Crossplane will
 not install _any_ CRDs for a package unless it can determine that _all_ CRDs can
 be installed. This guards against multiple Providers attempting to reconcile the
 same CRDs. Crossplane will also create a `ServiceAccount` with permissions to
 reconcile these CRDs and it will be assigned to the controller `Deployment`.
 
+The `spec.controller.image` fields specifies that the `Provider` desires for the
+controller `Deployment` to be created with the provided image. It is important
+to note that this image is separate from the package image itself. In the case
+above, it is an image containing the `provider-gcp` controller binary.
+
+The `spec.controller.permissionRequests` field allows a package author to
+request additional RBAC for the packaged controller. The controller's
+`ServiceAccount` will automatically give the controller permission to reconcile
+all types that its package installs, as well as `Secrets`, `ConfigMaps`, and
+`Events`. Any additional permissions must be explicitly requested.
+
+> Note that the Crossplane RBAC manager can be configured to reject permissions
+> for certain API groups. If a package requests permissions that Crossplane is
+> configured to reject, the package will fail to be installed. 
```

Comment:
```suggestion
> Note that the Crossplane RBAC manager must be configured to allow permissions
> for certain API groups. If a package requests permissions that Crossplane is not
> configured to allow, the package will fail to be installed. 
```

Reply:
Just flipping this phrasing since we require the user to pass a `ClusterRole` of permissions we allow, rather than permissions we reject.

Reply:
Ah great point! Thanks :)